src/arutils.py

```diff
@@ -239,12 +258,69 @@ def func_fit(x, y, func, deg, minv=None, maxv=None, w=None, **kwargs):
         return np.polynomial.chebyshev.chebfit(xv, y, deg, w=w)
     elif func == "bspline":
         return bspline_fit(x, y, order=deg, w=w, **kwargs)
+    elif func in ["gaussian"]:
+        # Guesses
+        if guesses is None:
+            mx = np.max(y)  # Could take simple stats near center
```

Comment:
Well, at present neither fit allows for a non-zero 'continuum'
in the parametrization.  But we might someday..
That said, I'm not liking the maximum either.  I've going 
to implement a more robust approach (I hope).